src/Installer/PackagesInstaller.php

```diff
@@ -99,19 +111,21 @@ public function __construct(
     public function install(): void
     {
         $type = $this->typeResolver->resolve();
-        if (!isset($this->mapping[$type])) {
-            return;
-        }
+        $projectTypePackages = $this->mapping[$type] ?? [];
+        $packagesToInstall = array_merge($projectTypePackages, $this->mapping[MappingResolver::DEFAULT_MAPPING_TYPE]);
```

Comment:
Suggest: Maybe the current `'name' => 'phpunit/phpunit'` should be moved to the key, so

```
MappingResolver::DEFAULT_MAPPING_TYPE => [
    'phpunit/phpunit' => [
                'version' => '@stable',
                'dev' => true,
   ]
]
```

so that project type settings can override the default settings (need a change in the argument order in array_merge too)